neo/framework/Common.cpp

```diff
@@ -3291,14 +3320,18 @@ bool idCommonLocal::GetAdditionalFunction(idCommon::FunctionType ft, idCommon::F
 			// don't set *out_userArg, this function takes no arguments
 			return true;
 
+		case idCommon::FT_UpdateDebugger:
+			*out_fnptr = (idCommon::FunctionPointer)updateDebugger;
+
```

Comment:
it might be a bit hacky, but if it turns out that you need to know whether the mod DLL supports debugging (`idGameEditExt`), you could set some global `bool` here (only mod DLLs implementing `idGameEditExt` will use `FT_UpdateDebugger`) and check that at appropriate places in the debugger server

Reply:
yeah thx. i forgot :P